src/cpl/nuopc/rof_comp_nuopc.F90

```diff
@@ -659,6 +647,18 @@ subroutine ModelAdvance(gcomp, rc)
        nlend = .false.
     endif
 
+    !--------------------------------
+    ! Unpack import state from mediator
+    !--------------------------------
+
+    call t_startf ('lc_mosart_import')
+
+    write(6,*)'DEBUG: calling mosart import'
```

Comment:
This should be removed, or put into a "if ( debug > 0 ) statement. Or the DEBUG string removed.